lib/datadog/core/environment/gc.rb

```diff
@@ -0,0 +1,18 @@
+module Datadog
+  module Core
+    module Environment
+      # Retrieves garbage collection statistics
+      module GC
+        module_function
+
+        def stat
+          ::GC.stat
+        end
+
+        def available?
+          defined?(::GC) && ::GC.respond_to?(:stat)
+        end
+      end
+    end
+  end
+end
```

Comment:
I guess out of scope for this PR, but I somewhat question the utility of this module (and to be honest, ClassCount as well).